gdnative-core/src/nativescript/init.rs

```diff
@@ -271,13 +282,21 @@ impl<C: NativeClass> ClassBuilder<C> {
     }
 
     #[inline]
-    pub fn add_method(&self, name: &str, method: ScriptMethodFn) {
+    pub fn add_method(&self, name: &str, method: ScriptMethodFn, rpc: &str) {
+        let rpc_mode = match rpc {
+            "remote" => RpcMode::Remote,
+            "remotesync" => RpcMode::RemoteSync,
+            "master" => RpcMode::Master,
+            "puppet" => RpcMode::Puppet,
+            "puppetsync" => RpcMode::PuppetSync,
+            "mastersync" => RpcMode::MasterSync,
+            _ => RpcMode::Disabled,
+        };
+
         self.add_method_advanced(ScriptMethod {
             name,
             method_ptr: Some(method),
-            attributes: ScriptMethodAttributes {
-                rpc_mode: RpcMode::Disabled,
-            },
+            attributes: ScriptMethodAttributes { rpc_mode: rpc_mode },
```

Comment:
`{ rpc_mode: rpc_mode }` can be simplified to `{ rpc_mode }`.

Reply:
Got it. Thanks!